packages/@dcl/inspector/src/lib/sdk/operations/add-asset/index.ts

```diff
@@ -80,6 +83,36 @@ export function addAsset(engine: IEngine) {
             values.set(componentName, { ...audioSource, src: audioSource.audioClipUrl.replace('{assetPath}', base) })
             break
           }
+          case CoreComponents.VIDEO_PLAYER: {
+            const videoPlayer = values.get(componentName) as PBVideoPlayer
+            values.set(componentName, { ...videoPlayer, src: videoPlayer.src.replace('{assetPath}', base) })
+            break
+          }
+          case CoreComponents.MATERIAL: {
+            const material = values.get(componentName) as PBMaterial
+            values.set(componentName, parseMaterial(base, material))
+            break
+          }
+          case CoreComponents.MESH_RENDERER: {
+            const mesh = values.get(componentName) as PBMeshRenderer
+            values.set(componentName, mesh)
+            break
+          }
+          case CoreComponents.VISIBILITY_COMPONENT: {
+            const visibility = values.get(componentName) as PBVisibilityComponent
+            values.set(componentName, visibility)
+            break
+          }
+          case CoreComponents.AUDIO_STREAM: {
+            const audioStream = values.get(componentName) as PBAudioStream
+            values.set(componentName, audioStream)
+            break
+          }
+          case CoreComponents.NFT_SHAPE: {
+            const nft = values.get(componentName) as PBNftShape
+            values.set(componentName, nft)
```

Comment:
the cases above `MESH_RENDERER`, `VISIBILITY_COMPONENT`, `AUDIO_STREAM` and `AUDIO_STREAM` are not making any modifications to the values, so they can be removed (they are getting the value and then setting it again without changes).

This section is only for making modifications to values.